manager/src/main/java/com/bakdata/quick/manager/k8s/KubernetesManagerClient.java

```diff
@@ -79,12 +80,19 @@ public Completable delete(final QuickResources quickResources) {
      * Deploys all the resources passed in.
      *
      * @param quickResources A quick resources object containing one or many kubernetes resources
-     * @return Completable or throws an exception if fail
+     * @return Returns a Completable or throws an exception if specific resources already exist
```

Comment:
No need to duplicate `return`:
```suggestion
     * @return The completable or an exception if the specific resources already exist
```